pkg/ccl/changefeedccl/protected_timestamps_test.go

```diff
@@ -1150,12 +1173,13 @@ func TestChangefeedPerTableProtectedTimestampProgression(t *testing.T) {
 							return errors.Newf("expected PTS record for table %d", tableID)
```

Comment:
Could we update this test to assert that the records all have different IDs as well?